custom_student_verification/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '1.2.4'
+__version__ = '1.3.5'
```

Comment:
@alfredchavez, shouldn't this be `1.3.0`?
cc: @giovannicimolin

Reply:
Yup, the new version should be `1.3.0`.